src/blob_gc.h

```diff
@@ -12,7 +12,8 @@ namespace titandb {
 // A BlobGC encapsulates information about a blob gc.
 class BlobGC {
  public:
-  BlobGC(std::vector<BlobFileMeta*>&& blob_files,
+  BlobGC(std::vector<BlobFileMeta*>&& gc_blob_files,
+         std::vector<BlobFileMeta*>&& fs_blob_files,
```

Comment:
We also need to modify construction in blob_gc_job_test.cc:188.

Reply:
fixed